### ISSUE TITLE
Test suite fails on Linux due to CR/LF line endings

### ISSUE THREAD

**Issue by oliverstory** [openCEMorg/openCEM#19]
The test suite failed on linux due to windows line endings in Sim2020.dat and Sim2025.dat

After running the test suite, and returning the failures posted below, I converted those two files to unix line endings and the tests then passed.

Options to solve:
- add a git attributes file to define how to convert line endings on git clone (this would also solve other line-ending issues): https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings

- change the test code, maybe like here: https://stackoverflow.com/questions/3280317/best-way-to-check-new-line-independent-identity-of-2-files-with-python/3280409#3280409

- change the wiki pointing this out

See test results below:

```
oliver@hakea$ pytest
........................................................................ [ 60%]
...sss........................FF
=================================== FAILURES ===================================
__________________________ test_multi_template_first ___________________________

    def test_multi_template_first():
        '''Tests generate first year template by comparing to known good result'''
        multi_sim = SolveTemplate(cfgfile='tests/testConfig.cfg')
        multi_sim.generateyeartemplate(multi_sim.Years[0], test=True)
>       assert filecmp.cmp(multi_sim.tmpdir + 'Sim2020.dat', 'tests/Sim2020.dat')
E       AssertionError: assert False
E        +  where False = <function cmp at 0x7f16a1025200>(('/tmp/tmph6vayhgm/' + 'Sim2020.dat'), 'tests/Sim2020.dat')
E        +    where <function cmp at 0x7f16a1025200> = filecmp.cmp
E        +    and   '/tmp/tmph6vayhgm/' = <cemo.multi.SolveTemplate object at 0x7f167ed3d710>.tmpdir

tests/test_multi.py:96: AssertionError
__________________________ test_multi_template_second __________________________

delete_sim2025_dat = None

    def test_multi_template_second(delete_sim2025_dat):
        '''Tests generate second (and later) year template by comparing to known good result'''
        multi_sim = SolveTemplate(cfgfile='tests/testConfig.cfg', tmpdir='')
        multi_sim.generateyeartemplate(multi_sim.Years[1], test=True)
>       assert filecmp.cmp(multi_sim.tmpdir + 'Sim2025.dat', 'tests/Sim2025.dat')
E       AssertionError: assert False
E        +  where False = <function cmp at 0x7f16a1025200>(('' + 'Sim2025.dat'), 'tests/Sim2025.dat')
E        +    where <function cmp at 0x7f16a1025200> = filecmp.cmp
E        +    and   '' = <cemo.multi.SolveTemplate object at 0x7f167d7ce550>.tmpdir

tests/test_multi.py:103: AssertionError
2 failed, 99 passed, 3 skipped in 25.07 seconds
```

**Comment by jotaigna** [comment]
Thank you for raising this, we will produce a fix shortly

**Comment by jotaigna** [comment]
Hi @oliverstory I am unable to reproduce this error with
- python 3.7 and 3.8 in Ubuntu or
- python 3.8 on a Windows machine. In both instances checked out with git.
- Dockerfile on linux environment
- Dockerfile on Windows environment

Is it possible for you to share some details regarding your setup?
Alternatively did you download a zip file of the repository or checked it out with git?

**Comment by oliverstory** [comment]
Hi Jose

You've prompted me to go digging into my git settings. It's been many years since I last used git. I have this set:

```
[core]
        autocrlf = true
```

So I think I've caused the problem myself on checkout!

In terms of the rest of my setup:

I am on Ubuntu 20.04. I checked out opencem with git per the wiki instructions: 
`git clone https://github.com/openCEMorg/openCEM.git`

I installed miniconda and installed requirements using the environment.yaml file.
